frontend/src/modules/Environments/views/EnvironmentEditForm.js

```diff
@@ -494,7 +497,7 @@ const EnvironmentEditForm = (props) => {
                                   </FormGroup>
                                 </Box>
                               )}
-                              {isModuleEnabled(ModuleNames.PIPELINES) && (
+                              {isModuleEnabled(ModuleNames.DATAPIPELINES) && (
```

Comment:
I am not sure I understood you, you mean as components created in the module folder that are imported into the environment view? My idea for this view was to define this particular block in a more abstract way. Instead of hardcoding the different environment modules, I would prefer to have some sort of logic that creates the blocks based on the information in the index files + config activation status of the modules. But looking at it now i think we can combine both approaches:

- iterate over the environment modules and call isModuleEnabled
- import the correspondent environment-module-component inside the condition of isModuleEnabled

Reply:
We can create a new issue for this particular problem + review if we can move some of the current code from `frontend/services` into `modules` given that we define the dependencies well

Reply:
> Instead of hardcoding the different environment modules, I would prefer to have some sort of logic that creates the blocks based on the information in the index files

This is a good idea.

> I will also propose to create another directory frontend/core/ where we place modules such as Environment. Wdyt? Does this make sense or is it against frontend design rules?

I believe it's a good idea to consider which worth it's own ticket. I'd like that we gather prons and cons for it ( + we could try to start writing ADR as we had discussed before)